app/app.go

```diff
@@ -703,6 +723,26 @@ func New(
 	app.ScopedTransferKeeper = scopedTransferKeeper
 	// this line is used by starport scaffolding # stargate/app/beforeInitReturn
 
+	// upgrade handler
+	plan0_7_0 := "v0.7.0"
+	app.UpgradeKeeper.SetUpgradeHandler(plan0_7_0, func(ctx sdk.Context, plan upgradetypes.Plan, fromVM module.VersionMap) (module.VersionMap, error) {
+		return app.mm.RunMigrations(ctx, app.configurator, fromVM)
```

Comment:
not sure if the ibc-go module version increased? https://github.com/cosmos/ibc-go/blob/main/modules/core/module.go#L164
there isn't much change between ibc-go 1 and 2, but not sure whether one shouldn't explicitly pass it in the upgrade handler so that the version map gets upgraded for the future

Reply:
I think the `fromVM` is loaded from the storage, and `RunMigrations` will migrate them to current version automatically, so it should work?

Reply:
if that's desirable with the fork to upstream migration later (I'm assuming when it adds the hooks or middleware, it'll be increased to 3?)

Reply:
> if that's desirable with the fork to upstream migration later (I'm assuming when it adds the hooks or middleware, it'll be increased to 3?)

do you mean the cronos module? I think if there's nothing to migrate in the storage, there's no need to increase the consensus version, right?

Reply:
I meant the ibc-go fork. ConsensusVersion in a module should be increased with any state-breaking changes, so hopefully it's fine